spec/workers/every_morning_worker_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'fugit'
+
+RSpec.describe Schedule::EveryMorningWorker, type: :worker do
+  let(:time) { (Time.zone.tomorrow + 8.hours).to_datetime }
+  let(:scheduled_job) { described_class.perform_at(time) }
+
+  describe 'testing worker' do
+    it 'is enqueued in the schedul queue' do
+      described_class.perform_async
+
+      assert_equal :schedule, described_class.queue
+    end
+
+    it 'goes into jobs array for testing environment' do
```

Comment:
still not sure what you're gonna test here

Reply:
to check whether the job actually goes to the testing environment or not, but since any other tests are passed, I think this test can be justified. I will remove it right away.